actions/web_actions.js

```diff
@@ -48,20 +24,14 @@ export const checkForImagesOnWeb = token => {
                 console.log('RESPONSE: checkForImagesOnWeb', resp.data.photos);
 
                 if (resp.data.photos) {
-                    // Todo - load Tags: {} with the data
-                    let photos = resp.data.photos ? resp.data.photos : null;
-
-                    photos = photos.map(photo => {
-                        photo.tags = {};
-                        return photo;
-                    });
+                    let photos = [];
+                    photos = resp.data.photos ? resp.data.photos : null;
 
-                    // if photos is null, pass empty array
                     dispatch({
-                        type: WEB_IMAGES,
+                        type: ADD_IMAGE,
                         payload: {
-                            count: resp.data.count,
-                            photos
+                            images: photos,
+                            type: 'WEB'
```

Comment:
lowercase is fine

Reply:
I was using same convention as we do for redux types..
Easier to identify that it's a type.

can change it to lowercase if u want.